File: ANDROID/Form/app/src/main/java/com/aramzy/form/MainActivity.java
package com.aramzy.form;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {
    private Button signInButton, signUpButton;
    private Intent intent;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        signInButton = (Button)findViewById(R.id.sign_in_button);
        signUpButton = (Button)findViewById(R.id.sign_up_button);

        signInButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                intent = new Intent(MainActivity.this ,SignInActivity.class);
                startActivity(intent);
            }
        });

        signUpButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                intent = new Intent(MainActivity.this ,SignUpActivity.class);
                startActivity(intent);
            }
        });
    }
}

File: ANDROID/MySQLiteFull/app/src/main/java/com/aramzy/mysqlitefull/MainActivity.kt
package com.aramzy.mysqlitefull

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Button

class MainActivity : AppCompatActivity() {

    lateinit var databaseBtn: Button
    lateinit var insertBtn: Button

    override fun onCreate(savedInstanceState: Bundle?) {

        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        databaseBtn = findViewById(R.id.databaseBtn)
        insertBtn   = findViewById(R.id.insertDB)

        databaseBtn.setOnClickListener(View.OnClickListener {

            val intent: Intent = Intent(this, DatabaseActivity::class.java)
            startActivity(intent)

        })

        insertBtn.setOnClickListener(View.OnClickListener {

            val intent: Intent = Intent(this, InsertActivity::class.java)
            startActivity(intent)

        })

    }

}